### ISSUE TITLE
Task - Applicant - Build Progress Bar component  for Application

### ISSUE THREAD

**Issue by tristan-orourke** [GCTC-NTGC/TalentCloud#2865]
# Description
Build a new progress bar for the application. This should hopefully reuse the progress bar component previously used for the JPB.

WAITING ON A FINAL DESIGN

**Comment by tristan-orourke** [comment]
Duplicate of #2653
